apps/backend/next.config.mjs

```diff
@@ -47,6 +47,10 @@ const withConfiguredSentryConfig = (nextConfig) =>
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  // optionally set output to "standalone" for Docker builds
+  // https://nextjs.org/docs/pages/api-reference/next-config-js/output
+  output: process.env.NEXT_CONFIG_OUTPUT,
```

Comment:
When should this *not* be standalone?

Reply:
If we deploy on vercel, we don't want it to be standalone

Reply:
Why not?

Reply:
For local dev and Vercel, setting output = standalone only increased build size and build time